cmd/operator-sdk/add/api.go

```diff
@@ -21,89 +21,257 @@ import (
 	"path/filepath"
 
 	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/internal/genutil"
-	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
+	apiflags "github.com/operator-framework/operator-sdk/internal/flags/apiflags"
 	"github.com/operator-framework/operator-sdk/internal/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/ansible"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/helm"
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+	"gopkg.in/yaml.v2"
+	"k8s.io/client-go/discovery"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
```

Comment:
fix the import order

Reply:
@camilamacedo86 I don't understand the order you want. When I resave this file goimports is sorting them the same way and it passes the linter's gofmt. So I would NOT reorder these.

Reply:
Hi @jmrodri, 

It is a standard for SKD and k8s projects.  The imports should be organized as:

statics
-- blank line
external
-- blank line
internal 

E.g : https://github.com/operator-framework/operator-sdk/blob/master/pkg/helm/controller/controller.go

Reply:
OH! yeah that was not obvious with your original comment :)